### DISCUSSION TITLE
[RFC] Stash doesn't find Videos on external Drive Linux

### DISCUSSION THREAD

**Discussion by trytofly94** [stashapp/stash#3606] [Q&A]
# Stash doesn't find Videos on external Drive Linux

### Scope
When I scan my folder on the external drive the videos don't show up. Running Linux Mint 21, drive is mounted on /mnt, permissions for user are given to the whole drive.

##  Long Form
Basically the title. I run stash on Linux Mint 21 and have my external harddrive mounted under /mnt. When I add said folder on the drive to my library and scan, no videos show up in my library. I checked the permission and /mnt is owned by root but /mnt/drive etc are owned by user with write permissions. Do I need to change the /mnt permissions to user. I don't want to break the system. I tried running Stash from the ext HD but no change.

**Comment by dingdongsolong4** [comment]
What user is stash running under? Are you running it from your own user account (ie manually from a terminal) or from a system service of some kind?

If stash it running under your own user account, then if you can access the folder and open files in it using a file manager then stash should be able to access it as well.

Are there any error messages in the stash logs (settings/logs)?
